### ISSUE TITLE
API: cEntity:HandleSpeedFromAttachee crashes the server

### ISSUE THREAD

**Issue by Seadragon91** [cuberite/cuberite#3662]
Server OS:        Windows
Commit id:        4d930b877cbfda32c5d6b7518d889bd6bdc24dea

### Actual behavior
Calling the function with the params, crashes the server.

### Steps to reproduce the behavior
1) Add this code to a plugin inside a command handler
2) Call the command
`cRoot:Get():GetDefaultWorld():ForEachEntity(function(a_Entity) a_Entity:HandleSpeedFromAttachee(1, 1) return true end)`

**Comment by NiLSPACE** [comment]
I don't think that method is even supposed to be exported to the API.

**Comment by Seadragon91** [comment]
Yes you are right, it's not listed on the api doc website, but it's in `APIDesc.lua`.
Then the fix is to remove it from export.
